packages/ui/src/histoire/histoire.setup.ts

```diff
@@ -0,0 +1,31 @@
+import { defineSetupVue3 } from '@histoire/plugin-vue'
+
+import demoIconAliases from '../vue-book/vuestic-config/demo-icon-aliases'
+import demoIconFonts from '../vue-book/vuestic-config/demo-icon-fonts'

```

Comment:
`vuestic-config` folder should be in `histoire` folder

Reply:
That'll be handled as part of https://github.com/epicmaxco/vuestic-ui/issues/2924

> If it works we replace vue-book with Histoire.

Right now vue-book is still functional and I don't see a good reason to duplicate these configs.